app/pathogen/arbovirus/wip/visualizations-section.tsx

```diff
@@ -29,19 +29,24 @@ export const VisualizationsSection = () => {
     ].includes(visualizationInfo.id));
 
     const renderVisualizationList = useCallback((visualizationList: Array<VisualizationInformation & {className: string}>) => {
-      return visualizationList.map((visualization, index) => (
-        <div key={visualization.id} className={cn(visualization.className, index != 0 ? 'mt-14' : undefined)}>
-          <div className="flex py-4">
-            <h3 className="w-full text-center text-lg">{visualization.displayName}</h3>
-            <button
-              onClick={() => router.push(`visualizations?visualization=${visualization.urlParameter}&referrerRoute=/pathogen/arbovirus/wip%3Fsection=${RedesignedArbovirusPageSectionUrlParam.visualizations}`)}
-              aria-label="See visualization in fullscreen"
-            >
-              <ZoomIn />
-            </button>
-          </div>
-          {visualization.renderVisualization()}
-        </div>
+      return visualizationList.map((visualizationInformation) => (
+        <RechartsVisualization
+          key={visualizationInformation.id}
+          visualizationInformation={visualizationInformation}
+          className={cn(visualizationInformation.className, 'pb-14')}
```

Comment:
Padding necessary for correct dimensions on downloaded image and to prevent the graphs from overlapping with one another on the website. This used to be margin and that worked fine but I need it as padding because I want it to be part of the downloaded region if that makes sense.